GOOUT/Controller/ForgotPasswordViewController.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  ForgotPasswordViewController.swift
+//  GOOUT
+//
+//  Created by GSM07 on 2021/09/02.
+//
+
+import UIKit
+import SnapKit
+import Then
+
+class ForgotPasswordViewController: UIViewController{
+    
+    var mainview = UIView()
+    var mainlabel = UILabel()
+    var passwordlabel = UILabel()
+    var sublable = UILabel()
+    var sublable2 = UILabel()
+    var emailtext = UITextField()
+    var checkbutton = UIButton()
+    let border = CALayer()
+    let bound = UIScreen.main.bounds
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+       configureUI()
+      
+     
+            
+        }
+        
+    
```

Comment:
```suggestion 
        }
        
```
공백들만 조금 신경써주면 더 좋은 코드가 될 수 있지 않을까요??🙇‍♀️